File: test/fixtures/node-axios/get_user_agent_alias.js
import axios from 'axios';

const response = await axios.get('http://localhost:28139/vc/moviesmagic', {
  params: {
    'p': '5',
    'pub': 'testmovie',
    'tkn': '817263812'
  },
  headers: {
    'x-msisdn': 'XXXXXXXXXXXXX',
    'user-agent': 'Mozilla Android6.1'
  }
});

File: test/fixtures/kotlin-ktor/get_user_agent_alias.kt
import io.ktor.client.HttpClient
import io.ktor.client.request.get
import io.ktor.client.request.header
import io.ktor.client.statement.bodyAsText
import io.ktor.http.isSuccess
import kotlinx.coroutines.runBlocking

val client = HttpClient()

runBlocking {
    val response = client.get("http://localhost:28139/vc/moviesmagic?p=5&pub=testmovie&tkn=817263812") {
        header("x-msisdn", "XXXXXXXXXXXXX")
        header("user-agent", "Mozilla Android6.1")
    }
    if (!response.status.isSuccess()) throw Exception("Request failed with status code ${response.status}")

    println(response.bodyAsText())
}
